lib/core/src/text/mod.rs

```diff
@@ -0,0 +1,288 @@
+use basegl_backend_webgl::{Context, compile_shader, link_program, Program, };
+
+use web_sys::{WebGlRenderingContext, WebGlBuffer, WebGlTexture};
+use crate::prelude::*;
+
+pub mod font;
+
+use font::FontRenderInfo;
+use crate::text::font::MsdfTexture;
+use js_sys::Float32Array;
+use crate::display::workspace::WorkspaceData;
+
+#[derive(Debug)]
+pub struct Color {
+    pub r : f32,
+    pub g : f32,
+    pub b : f32,
+    pub a : f32
+}
```

Comment:
We could make use of this struct in other places too. Why don't we expose it somewhere else?

Reply:
Moved to `lib.rs` for now (I haven't found a better place)